pkg/manager/manager_tasks.go

```diff
@@ -626,6 +639,54 @@ func (s *singleton) Clean() {
 	}()
 }
 
+func (s *singleton) MigrateHash() {
+	if s.Status.Status != Idle {
+		return
+	}
+	s.Status.SetStatus(Clean)
+	s.Status.indefiniteProgress()
+
+	qb := models.NewSceneQueryBuilder()
+
+	go func() {
+		defer s.returnToIdleState()
+
+		logger.Infof("Migrating generated files for current naming hash")
+
+		scenes, err := qb.All()
+		if err != nil {
+			logger.Errorf("failed to fetch list of scenes for cleaning")
+			return
+		}
+
+		var wg sync.WaitGroup
+		s.Status.Progress = 0
+		total := len(scenes)
+		useMD5 := config.IsUseMD5()
+
+		for i, scene := range scenes {
+			s.Status.setProgress(i, total)
+			if s.Status.stopping {
+				logger.Info("Stopping due to user request")
+				return
+			}
+
+			if scene == nil {
+				logger.Errorf("nil scene, skipping Clean")
```

Comment:
same as above for `Clean` 
as a side note `Clean` doesn't have a status message so nothing is printed in the status when doing the name migration